asus/zephyrus/ga402/default.nix

```diff
@@ -0,0 +1,20 @@
+{ ... }:
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/pc/laptop
```

Comment:
This just does nothing.

Reply:
oh why ? it does not apply their content ? 
It is done on other files like here https://github.com/NixOS/nixos-hardware/blob/master/asus/zephyrus/ga503/default.nix

Reply:
The module itself doesn't do anything on modern NixOS setups.